frontend/schemaCMS/src/routes/page/editPage/editPage.component.js

```diff
@@ -254,6 +270,40 @@ export const EditPage = ({
           </NextButton>
         </ModalActions>
       </Modal>
+      <Modal ariaHideApp={false} isOpen={leavingPageModalOpen} contentLabel="Confirm Leaving Page" style={modalStyles}>
```

Comment:
Content label should come from messages file too

Reply:
done